### PR TITLE
Benji/ios 594 notification deeplinks

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/Coordinator/MainCoordinator+Launch.swift [hunk 1]
```diff
@@ -26,6 +26,9 @@ extension MainCoordinator {
             let coordinator = RoomCoordinator(router: self.router, deepLink: self.deepLink)
             self.addChildAndStart(coordinator, finishedHandler: { (_) in})
             self.router.setRootModule(coordinator)
+            if let deepLink = deepLink {
```
Comment (Benjamindodgson) [inline Shared/Coordinator/MainCoordinator+Launch.swift]: forgot to add this back in.

File: iOS/Flows/Room/RoomHeaderView.swift [hunk 2]
```diff
@@ -31,16 +39,26 @@ class RoomHeaderView: BaseView {
...
-            self.focusCircle.set(backgroundColor: FocusStatus.focused.color)
-        }
+        self.update(status: User.current()?.focusStatus)
         
         self.addSubview(self.button)
         
         self.addSubview(self.roomNavButton)
         self.roomNavButton.configure(for: .outer)
+        
+        PeopleStore.shared.$personUpdated.mainSink { person in
```
Comment (Benjamindodgson) [inline iOS/Flows/Room/RoomHeaderView.swift]: Update the circle color when focus status changes.

File: iOS/UI/Shared/StackedPersonView.swift [hunk 3]
```diff
@@ -56,7 +56,6 @@ class StackedPersonView: BaseView {
         var count: Int = 0
         self.subviews.forEach { view in
             if let personView = view as? BorderedPersonView {
-                personView.pulseLayer.borderWidth = 1
```
Comment (Benjamindodgson) [inline iOS/UI/Shared/StackedPersonView.swift]: Just making it consistent

File: iOS/UI/Swipeable Input/ExpressionView.swift [hunk 4]
```diff
@@ -21,7 +21,7 @@ class ExpressionView: BaseView {
         self.set(backgroundColor: .clear)
         
         self.addSubview(self.imageView)
-        self.imageView.tintColor = ThemeColor.white.color.withAlphaComponent(0.3)
+        self.imageView.tintColor = ThemeColor.B5.color.withAlphaComponent(0.3)
```
Comment (Benjamindodgson) [inline iOS/UI/Swipeable Input/ExpressionView.swift]: This works for light/dark mode.

File: iOS/Flows/Room/RoomHeaderView.swift (hunk 2 above)
Comment (MartinJibber) [reply iOS/Flows/Room/RoomHeaderView.swift]: We'll need an unowned reference or else this is a memory leak

**Review by MartinJibber — CHANGES_REQUESTED** [review]
One change needed

**Pull request merged**